contingency/contingency-api/src/main/java/com/powsybl/contingency/Contingency.java

```diff
@@ -30,19 +30,30 @@ public class Contingency extends AbstractExtendable<Contingency> {
 
     private final List<ContingencyElement> elements;
 
+    private final Set<String> elementsNotFound;
```

Comment:
Do we store elementsNotFound in Contingency while it is specific to IdBasedContingencyList ?
It could be a method of IdBasedContingencyList only ?

Reply:
Contingency has no subclass, it is the same regardless of the type. We have different types of ContingencyList (which implements the ContingencyList interface). And yes, the not found elements are only for IdentifierContingencyList, that's why I added the getAllContingencies method (it returns all contingencies without filtering) to IdentifierContingencyList only.